package.json

```diff
@@ -109,7 +109,7 @@
   "scripts": {
     "test": "jest",
     "dev": "sb dev -p 6006",
-    "build": "microbundle --jsx React.createElement --jsxFragment React.Fragment",
+    "build": "export NODE_OPTIONS=--max_old_space_size=4096 && microbundle --jsx React.createElement --jsxFragment React.Fragment --target node",
```

Comment:
Are u sure we need "--target node" ?

![image](https://github.com/kbss-cvut/s-forms/assets/3962237/ec77d01b-cc72-447f-a539-e57eb80f672a)

Reply:
Yes agree, I did not pay attention to it but you are right